src/averageAges.js

```diff
@@ -14,14 +14,34 @@
  *
  * @return {number}
  */
+
+function getAverageAge(male) {
```

Comment:
let's fix it)

<img width="168" alt="image" src="https://user-images.githubusercontent.com/86183007/224638397-65fbb44e-9d59-440b-9649-76c267f5e9f4.png">